src/rules/exports-subpaths-style.ts

```diff
@@ -37,7 +37,7 @@ export const rule = createRule({
 				fix(fixer) {
 					const valueText = context.sourceCode.getText(value);
 					const fixedValue = JSON.stringify(
-						// eslint-disable-next-line @typescript-eslint/no-unsafe-assignment -- JSON.parse required for wrapping
+						// eslint-disable-next-line ts/no-unsafe-assignment -- JSON.parse required for wrapping
```

Comment:
eslint-config-un has `ts` plugin prefix by default for `typescript-eslint` (can be changed)